package.json

```
@@ -5,6 +5,7 @@
     "main": "lib/index",
     "source": "src/index",
     "react-native": "src/index",
+    "types": "./src/index.d.ts",
```

We will ship package with "built files" (lib folder created by `yarn build`). In lib folder there is already `index.d.ts`, so we can change it to `lib/index.d.ts`
```suggestion
    "types": "lib/index.d.ts",
```